rpc/link.c

```diff
@@ -61,7 +62,7 @@ struct rpc_link_state_transition {
 	const char *rlst_st_desc;
 };
 
-typedef void (*rpc_link_cb_t)(struct m0_rpc_link*, m0_time_t, struct m0_clink*);
+typedef void (*rpc_link_cb_t)(struct m0_rpc_link*, m0_time_t, struct m0_clink*, struct m0_co_op*);
```

Comment:
No more than 80 chars in a line.

Reply:
I have removed this patch (see my comment above). So that, I guess we can consider that it has been fixed.